encode_test.go

```diff
@@ -95,17 +115,17 @@ func toFromJSON(t *testing.T, in interface{}, recvr interface{}) {
 
 func TestNilEncodings(t *testing.T) {
 	// make sure sigs are okay with nil
-	a, b := SignatureS{}, SignatureS{}
+	a, b := Signature{}, Signature{}
```

Comment:
Comments are fine:

`DO NOT USE this interface. It is public by necessity but should never be used directly outside of this package. Only use the PubKey, never the PubKeyInner`